### ISSUE TITLE
Paging `people educated` to disk space.

### ISSUE THREAD

**Issue by ChrisCates** [theothequant/reddit_bot.py#2]
Assuming your database of people educated grows over time... 
It may exceed the RAM on your computer.

It may be a good idea to future-proof your codebase by storing and accessing people educated without loading the entire list in memory.

In this context, a tool like `redis`, `leveldb` or `memcache` may be your best bet.

**Comment by theothequant** [comment]
Yeah that makes sense. storing it in memory will not be the most efficient way to do that.
